### ISSUE TITLE
Bitcode support?

### ISSUE THREAD

**Issue by stensmir** [cloudpayments/CloudPayments_iOSSDKDemo#3]
Сейчас приходится отключать только ради вашего SDK.

**Comment by anatoliykant** [comment]
Такая же ситуация, планируете ли решить этот вопрос?!

**Comment by NursultanGalymzhan** [comment]
Большая просьба добавить поддержку bitcode!

**Comment by YauheniYarotski** [comment]
Решил значит проинтегрировать проект, и наступил сразу на 3 ошибки, которые описанные тут. Причем вопросы решаются за 1-2 дня и будет счастие разработчикам. Какое то прям не уважение. Что, теперь во всем проекте отключать биткод из-за то что клауд не может на 2 дня найти iOS разработчика?????

**Comment by a-ignatov-cp** [comment]
Добрый день! Извините за неудобства, в настоящее время ведется разработка новой версии CloudPayments SDK для iOS, в которой будет поддержка bitcode, а для распространения будет использоваться  cocoapods, так же исправлены и другие недостатки.

**Comment by Foboz** [comment]
Какие-нибудь новости по новому SDK? Могли бы просто старый SDK для начала обновить

**Comment by a-ignatov-cp** [comment]
Добрый день! Новый SDK решили делать в качестве примера с открытым исходным кодом, чтобы каждый мог адаптировать под собственные нужды, он почти готов. Сейчас чтобы не задерживать Вас могу предоставить Вам, исходные файлы данного SDK, Вы просто добавите их в свой проект на Objective-C или Swift(используя файл моста). Напишите мне: a.ignatov@cloudpayments.ru

**Comment by gazaret** [comment]
@ign4 Есть новости по SDK?

**Comment by ef1rspb** [comment]
@gazaret Смотри здесь https://github.com/cloudpayments/CloudPayments_iOSCheckout

**Comment by gazaret** [comment]
@ef1rspb Checkout является заменой sdk (ios и android)?

**Comment by ef1rspb** [comment]
@gazaret да, под ios точно. Там еще рефакторить и рефакторить, но уже работает.
Посмотри пример, но не используй его в своем проекте. тебе нужны только Card.h & Card.m.

**Comment by gazaret** [comment]
@ef1rspb спасибо!
